synapse/config/server.py

```diff
@@ -1168,6 +1181,22 @@ def generate_config_section(
         #
         #allow_per_room_profiles: false
 
+        # The largest allowed size for a user avatar. If not defined, no
+        # restriction will be imposed.
```

Comment:
My initial question was if this was bytes or pixel size -- from the value being `10M` I see it is bytes. Not sure it is worth clarifying.